.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           command: |
             . venv/bin/activate
             export DJANGO_SETTINGS_MODULE=a_socials.settings.dev
-            python server/manage.py test --snapshot-update server/api/tests/ server/graphql_schemas/tests/
+            python server/manage.py test --snapshot-update server/api/tests/ server/graphql_schemas/tests/ --keepdb
```

Comment:
This file should not be part of this PR. Please remove it @m-asiimwe

Reply:
noted